packages/frontend/tests/acceptance/chat/privately-with/-acceptance-test.ts

```diff
@@ -22,6 +22,14 @@ module('Acceptance | Chat | Privately With', function(hooks) {
   setupApplicationTest(hooks);
   clearLocalStorage(hooks);
 
+  module('is not logged in', function(hooks) {
+    test('document.title is unchanged', async function(assert){
+      var titleBefore = document.title;
```

Comment:
this test doesn't visit any pages / do actions, so `await settled()` maybe doesn't do anything?

What is the expected pre-login title?

Reply:
I believe it is currently just "emberclear" but I didn't want to code that in. Would the site for not logged in still be /chat/privately-with or should it just be the root emberclear site?

Reply:
emberclear is static in every language, so using it is fine. but also, you can test strings via i18n, like:

`this.owner.lookup('service:i18n').t('appname')` -- this is defined in en.yml.
 - this is the preferred (but not yet widely adopted way of testing string values)

 > Would the site for not logged in still be /chat/privately-with or should it just be the root emberclear site?

`/chat/privately-with` is not visitable when logged out, a redirect will occur to `https://emberclear.io/setup/new` (which there are tests for, I think)

Reply:
> `/chat/privately-with` is not visitable when logged out, a redirect will occur to `https://emberclear.io/setup/new` (which there are tests for, I think)

Will that redirect be done with the `await settled()`?

Reply:
maybe?

if you were to `visit` without `await`ing, then that's when you'd want to `await settled()`